inference-engine/src/mkldnn_plugin/nodes/mkldnn_strided_slice_node.cpp

```diff
@@ -56,53 +48,27 @@ bool MKLDNNStridedSliceNode::isSupportedOperation(const std::shared_ptr<const ng
     return true;
 }
 
-MKLDNNStridedSliceNode::MKLDNNStridedSliceNode(const std::shared_ptr<ngraph::Node>& op, const mkldnn::engine& eng, MKLDNNWeightsSharing::Ptr &cache) :
+MKLDNNStridedSliceNode::MKLDNNStridedSliceNode(const std::shared_ptr<ov::Node>& op, const mkldnn::engine& eng, MKLDNNWeightsSharing::Ptr &cache) :
         MKLDNNNode(op, eng, cache) {
     std::string errorMessage;
     if (!isSupportedOperation(op, errorMessage)) {
         IE_THROW(NotImplemented) << errorMessage;
     }
-    auto ss = ov::as_type_ptr<const ngraph::opset1::StridedSlice>(op);
-    if (inputShapes.size() != 3 && inputShapes.size() != 4) {
+    if (inputShapes.size() < 3 && inputShapes.size() > 5) {
```

Comment:
Should we check `inputShapes.size() > 4` for StridedSlice?

Reply:
Done.